manifests/x/Xanthus58/PortalStillAlive/1.0/Xanthus58.PortalStillAlive.installer.yaml

```diff
@@ -0,0 +1,20 @@
+# Created with YamlCreate.ps1 v2.1.1 $debug=NVS1.5-1-19041-1320
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Xanthus58.PortalStillAlive
+PackageVersion: "1.0"
+MinimumOSVersion: 10.0.0.0
+InstallerType: inno
+UpgradeBehavior: install
+ReleaseDate: 2022-03-20
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/Xanthus58/Portal-Still-Alive/releases/download/1.0/Still.Alive.exe
+  InstallerSha256: 861787E0517EF4E2FACD46178C647FC9855F38C97839BFA3A1C132B9494B894B
+  AppsAndFeaturesEntries:
+    - DisplayName: Still Alive version 1.0
+      DisplayVersion: "1.0"
+      Publisher: Xanthus58
+      ProductCode: "{3FB48396-A497-4D5A-85FE-E92C994D7F59}_is1"
```

Comment:
Hmm, i think we should be careful with these fields (only use them when it requires manual updating) because neither YamlCreate or WinGetCreate can update the fields, which then would break [winget-pkgs-automation](https://github.com/vedantmgoyal2009/vedantmgoyal2009/tree/main/winget-pkgs-automation)

Reply:
With the way Vedant has it set up, he can have custom scripts on a per-package basis. This means he can grep the installer file for the DisplayName and DisplayVersion properties and update them